### ISSUE TITLE
Mysterious maximum number of iteration error

### ISSUE THREAD

**Issue by antonkulaga** [coursier/coursier#593]
In one of my projects ( https://github.com/antonkulaga/CRISPR ) I get totally non informative coursierResolutions exception when I update coursier version to 1.0.0-RC6 :
```
[info] Loading project definition from /home/antonkulaga/CRISPR/project
[info] Set current project to crispr (in build file:/home/antonkulaga/CRISPR/)
> compile
[trace] Stack trace suppressed: run last crisprJVM/*:coursierResolutions for the full output.
[error] (crisprJVM/*:coursierResolutions) coursier.ResolutionException: Maximum number of iteration of dependency resolution reached
[error] Total time: 6 s, completed Jun 26, 2017 6:22:20 PM
> last crisprJVM/*:coursierResolutions 
coursier.ResolutionException: Maximum number of iteration of dependency resolution reached
[error] (crisprJVM/*:coursierResolutions) coursier.ResolutionException: Maximum number of iteration of dependency resolution reached
```
From there I have no idea which dependency is missing.

**Comment by antonkulaga** [comment]
Looks like:
coursierMaxIterations := 200
fixes the problem. That means that defaults are really bad in coursier, because the project is quite simple

**Comment by alexarchambault** [comment]
I ran into this too, it originates from the changes from RC5 to RC6. I'm about to push a reworked version of them, that fixes the issue for me.

That said, the default number of iterations should be increased in the sbt plugin, from 50 to 100 like in the CLI say (PR?)

**Comment by alexarchambault** [comment]
This should be fixed in `1.0.0-RC8`. Re-open if you still run into this issue.
